docs/root/start/install.rst

```diff
@@ -10,51 +10,31 @@ If you are :ref:`installing on Mac OSX <start_install_macosx>`, you can install
 Once you have installed Envoy, check out the :ref:`quick start <start_quick_start>` guide for more information on
 getting your Envoy proxy up and running.
 
-Install Envoy on Debian GNU/Linux
+Install Envoy on Debian GNU/Linux or Ubuntu Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can `install Envoy on Debian <https://www.getenvoy.io/install/envoy/debian/>`_
-using `Get Envoy <https://www.getenvoy.io/>`__ until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
+You can install Envoy on Debian or Ubuntu using `Get Envoy <https://cloudsmith.io/~tetrate/repos/getenvoy-deb-stable/setup/#formats-deb>`__
+until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
 
 .. code-block:: console
 
-   $ sudo apt update
-   $ sudo apt install apt-transport-https ca-certificates curl gnupg2 software-properties-common
-   $ curl -sL 'https://getenvoy.io/gpg' | sudo gpg --dearmor -o /usr/share/keyrings/getenvoy-keyring.gpg
-   # Verify the keyring - this should yield "OK"
-   $ echo 1a2f6152efc6cc39e384fb869cdf3cc3e4e1ac68f4ad8f8f114a7c58bb0bea01 /usr/share/keyrings/getenvoy-keyring.gpg | sha256sum --check
-   $ echo "deb [arch=amd64 signed-by=/usr/share/keyrings/getenvoy-keyring.gpg] https://dl.bintray.com/tetrate/getenvoy-deb $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/getenvoy.list
-   $ sudo apt update
-   $ sudo apt install getenvoy-envoy
-
-Install Envoy on Ubuntu Linux
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You can `install Envoy on Ubuntu <https://www.getenvoy.io/install/envoy/ubuntu/>`_
-using `Get Envoy <https://www.getenvoy.io/>`__ until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
-
-.. code-block:: console
-
-   $ sudo apt update
-   $ sudo apt install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
-   $ curl -sL 'https://getenvoy.io/gpg' | sudo gpg --dearmor -o /usr/share/keyrings/getenvoy-keyring.gpg
-   # Verify the keyring - this should yield "OK"
-   $ echo 1a2f6152efc6cc39e384fb869cdf3cc3e4e1ac68f4ad8f8f114a7c58bb0bea01 /usr/share/keyrings/getenvoy-keyring.gpg | sha256sum --check
-   $ echo "deb [arch=amd64 signed-by=/usr/share/keyrings/getenvoy-keyring.gpg] https://dl.bintray.com/tetrate/getenvoy-deb $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/getenvoy.list
-   $ sudo apt update
-   $ sudo apt install -y getenvoy-envoy
+   $ curl -1sLf \
+       'https://deb.dl.getenvoy.io/public/setup.deb.sh' \
+       | sudo -E bash
```

Comment:
not sure about this - im very reluctant to run a script from the interwebs as root

looking at the script it seems to require a lot of stuff (eg python) that really shouldnt be necessary to install a deb

Reply:
sure I'll inline the instructions from the cloud smith site